### ISSUE TITLE
[Issue] CLI ignores path params in request-urls

### ISSUE THREAD

**Issue by chrisnagel** [usebruno/bruno#2506]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Thanks to the very useful feature #484 from @game5413, it is now possible to define and use path parmeters.
The feature works as expected in the GUI version of Bruno.

The path parameters are apparently not taken into account in the CLI: A request sent via _bru run_ currently still contains the colon placeholders instead of the defined parameter values.

The issue can be reproduced with V1.19.0 (GUI) and V1.12.0 (CLI).

### .bru file to reproduce the bug

```
meta {
  name: demo
  type: http
  seq: 2
}

get {
  url: https://jsonplaceholder.typicode.com/todos/:num?limit=3
  body: none
  auth: none
}

params:query {
  limit: 3
}

params:path {
  num: 1
}

script:post-response {
  console.log("Request-Url: " + req.url);
}

```

### Screenshots/Live demo link

![image](https://github.com/usebruno/bruno/assets/5313051/4ebafced-d5ec-4724-9155-a6c53460d871)
*Request-Url sent by Bruno GUI*

![image](https://github.com/usebruno/bruno/assets/5313051/b48f316b-0a16-4a2e-ada8-1360461b6180)
*Request-Url sent by Bruno CLI*

**Comment by fgrande** [comment]
I made a Fix for this, I can still reproduce the bug on bru v1.20.1 (see attached zipped bru file)
When run on BrunoApp everything is ok, on bruno CLI it raise an error (since params on URL are not interpolated).
Going to attach a PR
[TestParams.bru.zip](https://github.com/user-attachments/files/16132678/TestParams.bru.zip)

**Comment by chrisnagel** [comment]
Bug is solved since #2582 - thank you, @fgrande
